crates/router/src/core/payouts/helpers.rs

```diff
@@ -20,12 +19,12 @@ use crate::{
 
 pub async fn make_payout_method_data<'a>(
     state: &'a AppState,
-    request: &api::PayoutCreateRequest,
+    payout_method_data: &Option<api::PayoutMethodData>,
```

Comment:
are you getting this from request when you are passing it to this function ?

Reply:
Yes, multiple flows needs to use this function and pmd is not present in every request, so we're directly passing the payout_method_data